drivers/osdp/Kconfig.pd

```diff
@@ -0,0 +1,217 @@
+#
+# Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config OSDP_NUM_CONNECTED_PD
+	int
+	default 1
+	help
+	  In PD mode, number of connected PDs is is always 1 and cannot
+	  be configured.
+
+config OSDP_PD_ADDRESS
+	int "PD Address"
```

Comment:
I suggest `Peripheral Device Address` here. Could this setting be under "PD ID Information menu", in which case having "PD Address" would look ok.

Reply:
Changed it to `Peripheral Device Address`. The specification calls these group of constants as "PD ID" so its better to leave grouped. Besides, PD address is runtime configurable along with baud rate by the CP.